.changeset/gorgeous-snakes-itch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hyperlane-xyz/cli': minor
+---
+
+Adding a command for registering/deregistering operator to Hyperlane AVS
```

Comment:
Nit: Past tense is most common in release notes, then present tense ("Add") but present participle tense like this is atypical